### PR TITLE
chore: Append users' src dir name to sys.path while model server starts for CPR.

### PR DESCRIPTION
**Pull request by abcdefgs0324**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: google/cloud/aiplatform/prediction/local_model.py [hunk 1]
```diff
@@ -251,6 +252,7 @@ def build_cpr_model(
                 handler, src_dir
             )
         environment_variables = {
+            "CPR_USER_DIR_NAME": Path(src_dir).name,
```
Comment (kthytang) [inline google/cloud/aiplatform/prediction/local_model.py:255]: Why do we need to do Path().name here?

File: google/cloud/aiplatform/prediction/model_server.py [hunk 2]
```diff
@@ -186,6 +187,10 @@ def set_number_of_workers_from_env() -> None:
 
 
 if __name__ == "__main__":
+    cpr_user_dir_name = os.getenv("CPR_USER_DIR_NAME")
+    if cpr_user_dir_name:
+        sys.path.append(os.path.join(os.getcwd(), cpr_user_dir_name))
```
Comment (kthytang) [inline google/cloud/aiplatform/prediction/model_server.py:192]: Can we insert at the beginning of sys.path? So it searches the user's modules first.

**Review by kthytang — CHANGES_REQUESTED** [review]

File: google/cloud/aiplatform/prediction/local_model.py (hunk 1 above)
Comment (abcdefgs0324) [reply google/cloud/aiplatform/prediction/local_model.py:255]: If users provide "tmp/src_dir", we check all the necessary code (predictor, handler, and requirements.txt) should be in "tmp/src_dir" and copy "src_dir" to the workdir in the images.

File: google/cloud/aiplatform/prediction/model_server.py (hunk 2 above)
Comment (abcdefgs0324) [reply google/cloud/aiplatform/prediction/model_server.py]: Done.

File: google/cloud/aiplatform/prediction/local_model.py (hunk 1 above)
Comment (kthytang) [reply google/cloud/aiplatform/prediction/local_model.py:255]: Ah, understood, thanks.

**Review by kthytang — APPROVED** [review]

**Review by SinaChavoshi — APPROVED** [review]
lgtm from metadata side.

**Pull request merged**